src/libraries/System.Text.Json/tests/Serialization/CustomConverterTests/CustomConverterTests.Dynamic.Sample.Tests.cs

```diff
@@ -26,14 +26,17 @@ private enum MyCustomEnum
         public static void VerifyPrimitives()
         {
             var options = new JsonSerializerOptions();
-            options.EnableDynamicTypes();
+
+            // To prevent calling the actual EnableDynamicTypes() just use the extension class directly
```

Comment:
`the actual EnableDynamicTypes()`

Is this comment stale? Seems like `JsonUnknownTypeHandling` took over this function.

Reply:
Thanks. Yes it was stale because EnableDynamicTypes() was an earlier iteration to avoid the reference to System.Linq.Expressions. I was able to revert the whole file.